spec/requests/review_pages_spec.rb

```diff
@@ -242,7 +242,7 @@
          review.review_date.should == Date.new(2020, 6, 7)
          review.feedback_deadline.should == Date.new(2020, 01, 21)
          page.should have_selector('.flash-success',
-          text: 'Review was updated. Your coach and invitees have been notified of the new deadline.')
+          text: 'Review was updated. Your invitees have been notified of the new feedback deadline.')
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.